ethpm/backends/http.py

```diff
@@ -0,0 +1,33 @@
+import requests
+
+from ethpm.backends.base import BaseURIBackend
+from ethpm.constants import RAW_GITHUB_AUTHORITY
+from ethpm.utils.uri import is_valid_github_uri
+from ethpm.validation import validate_uri_contents
+
+
+class GithubOverHTTPSBackend(BaseURIBackend):
+    """
+    Base class for all URIs pointing to a content-addressed Github URI.
+    """
+
+    def can_resolve_uri(self, uri: str) -> bool:
+        return is_valid_github_uri(uri)
+
+    def can_translate_uri(self, uri: str) -> bool:
+        """
+        GithubOverHTTPSBackend uri's must resolve to a valid manifest,
+        and cannot translate to another content-addressed URI.
+        """
+        return False
+
+    def fetch_uri_contents(self, uri: str) -> bytes:
+        http_uri, validation_hash = uri.split("#")
```

Comment:
You should use `urllib.parse.urlpase` and pull the `fragment` from that return value.